Content.Client/Examine/ExamineSystem.cs

```diff
@@ -50,13 +51,22 @@ public override void Initialize()
 
             SubscribeNetworkEvent<ExamineSystemMessages.ExamineInfoResponseMessage>(OnExamineInfoResponse);
 
+            SubscribeLocalEvent<DroppedEvent>(OnExaminedItemDropped);
+
             CommandBinds.Builder
                 .Bind(ContentKeyFunctions.ExamineEntity, new PointerInputCmdHandler(HandleExamine, outsidePrediction: true))
                 .Register<ExamineSystem>();
 
             _idCounter = 0;
         }
 
+        private void OnExaminedItemDropped(DroppedEvent ev)
+        {
+            if (!ev.User.Valid)
+                return;
```

Comment:
use iplayermanager to check that the person dropping the item is the local player.